app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -53,6 +53,51 @@ def self.description
     @description ||= ManageIQ::Providers::Kubevirt::Constants::PRODUCT
   end
 
+  def self.params_for_create
+    @params_for_create ||= {
+      :title  => "Configure Kubevirt",
+      :fields => [
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.server",
+          :label      => "Hostname",
+          :isRequired => true,
+          :validate   => [{:type => "required-validator"}]
+        },
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.port",
+          :type       => "number",
+          :isRequired => true,
+          :validate   => [
+            {:type => "required-validator"},
+            {:type             => "validatorTypes.MIN_NUMBER_VALUE",
+             :includeThreshold => true,
+             :value            => 1
+            },
+            {:type             => "validatorTypes.MAX_NUMBER_VALUE",
+             :includeThreshold => true,
+             :value            => 65535
+            }
+          ]
+        },
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.token",
+          :label      => "Token",
+          :type       => "password",
+          :isRequired => true,
+          :validate   => [{:type => "required-validator"}]
+        }
+      ]
+    }.freeze
+  end
+
+  def self.verify_credentials(args)
+    server, port, token = args.dig("endpoints", "default")&.values_at("server", "port", "token")
+    !!raw_connect(:server => server, :port => port, :token => token)
```

Comment:
Super minor but if the incoming hash already has the keys...this can be simplified to just use slice.

```ruby
!!raw_connect(args.dig("endpoints", "default")&.slice("server", "port", "token").symbolize_keys)
```

Reply:
Also, interestingly, the docs for raw_connect say it takes verify_ssl and ca_certs, but then never uses them? https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/149/files#diff-ff17b05a7c525eec833f0ddb0134a0d9R109-R111

Reply:
Yeah I couldn't tell if that was an oversight or not so I left the description for now.